README.md

```diff
@@ -19,56 +18,115 @@ Requirements
 
 1. Get the source code:
 
-       git clone https://github.com/wikimedia/ws-export.git
-       cd tool
+   ```console
+   git clone https://github.com/wikimedia/ws-export.git
+   cd ws-export
+   ```
 
 2. Install dependencies:
 
-       composer install --no-dev
+   ```console
+   composer install
+   ```
 
-   Then create a `.env.local` file.
+   A few PHP extensions are required; if these are not installed, Composer will let you know
+   and you will need to install them with your operating system's package manager.
+   For example, in Debian-based Linux distributions:
 
-   * In order to export to PDF, plain text, RTF, or Mobi formats
-     you should also install [Calibre](https://calibre-ebook.com)
-     so that the tool can use the `ebook-convert` command.
-   * To validate exported ebooks (with the `./bin/console app:check` command),
-     you should also install
-     [epubcheck](https://github.com/w3c/epubcheck).
-     If it's not installed at `/usr/bin/epubcheck` then
-     set the `EPUBCHECK_JAR` environment variable.
-   * To run the integration tests, also install the `fonts-linuxlibertine` package.
+   ```console
+   apt install php-sqlite3 php-zip php-curl
+   ```
 
-3. Create a mysql database and database user
-   and add these details to `.env.local`.
+   Then create a `.env.local` file:
 
-4. Create the database with `./bin/console doctrine:database:create`
+   ```console
+   cp .env .env.local
+   ```
 
-5. Run the migrations with `./bin/console doctrine:migrations:migrate`
+   Edit `.env.local` and set `APP_ENV=dev` and `APP_SECRET` to a random string of your choice.
 
-6. This tool uses the [Toolforge Bundle](https://github.com/wikimedia/ToolforgeBundle), and it connects to [multiple databases](https://github.com/wikimedia/ToolforgeBundle#replicas-connection-manager).
-  * Set replicas credentials in the `.env.local` file.
+3. [Install Symfony CLI](https://symfony.com/download#step-1-install-symfony-cli)
+   and start the development web server with:
 
-  * Establish an SSH tunnel to the replicas (only necessary on local environments)
-```bash
-$ ./bin/console toolforge:ssh
-```
-  * Bind address for docker enviroments
-```bash
-$ ./bin/console toolforge:ssh --bind-address=0.0.0.0
-```
+   ```console
+   symfony server:start -d
+   ```
+
+4. Open your web browser to e.g. <http://localhost:8000>
+   and the basic operations should be working with the following requirements:
+   * you can only export to EPUB format; and
+   * you must check the 'Exclude credits' option,
+     to avoid querying the database for usernames.
+
+   Continue on for setting up more exporting and development options.
+
+5. Install optional dependencies:
+
+   **Calibre:**
+   In order to export to PDF, plain text, RTF, or Mobi formats
+   you must install [Calibre](https://calibre-ebook.com)
+   so that the tool can use the `ebook-convert` command.
+
+   **epubcheck:**
+   To validate exported ebooks (with the `./bin/console app:check` command),
```

Comment:
```suggestion
   To validate exported ebooks (with the [`./bin/console app:check` command](#appcheck)),
```

By the way, if the `./bin/console` is needed to run the command, why isn't it included in the "CLI Usage" section below? :thinking: